src/main/java/com/ericsson/ei/notifications/HttpRequest.java

```diff
@@ -210,4 +223,7 @@ private void addJenkinsCrumbData(JsonNode crumb) {
         }
     }
 
+    private boolean checkIfPasswordEncrypted(final String password) {
+        return (password.startsWith("ENC(") && password.endsWith(")"));
```

Comment:
Don't we have this method in TextFormatter class?

Reply:
I moved it into that class. I changed name on it and made it static as well.

Reply:
ok, but the implementation seems to be in this HTTPRequest class,, shouldn't this be in shared method be in TextFormatted? Or am I seeing an old version of HTTPRequest class?

Reply:
Ok, see now the changes.